core/bessctl.cc

```diff
@@ -189,6 +190,7 @@ static ::Port* create_port(const std::string& name, const PortBuilder& driver,
   p->queue_size[PACKET_DIR_INC] = size_inc_q;
   p->queue_size[PACKET_DIR_OUT] = size_out_q;
 
+  ctx.SetNonWorker();
```

Comment:
Agreed. I can't find any knobs in gRPC for that though. @barath Did you have any experience?